OpenOversight/app/main/forms.py

```diff
@@ -17,6 +17,14 @@
 import datetime
 import re
 
+# Normalizes the "not sure" option to what it needs to be when writing to the database.
+# Note this should only be used for forms which save a record to the DB--not those that
+# are used to look up existing records.
+db_genders = list(GENDER_CHOICES)
+for index, choice in enumerate(db_genders):
+    if choice == ('Not Sure', 'Not Sure'):
+        db_genders[index] = (None, 'Not Sure')  # type: ignore
```

Comment:
I added the type: ignore to get flake8 to stop complaining about this line with "Incompatible types in assignment" -- we do in fact want None instead of a string for this one particular instance.